std/functional.d

```diff
@@ -36,13 +36,13 @@ $(TR $(TH Function Name) $(TH Description)
         $(TD Creates a function that binds the first argument of a given function
         to a given value.
     ))
-    $(TR $(TD $(LREF reverseArgs)), $(LREF binaryReverseArgs)
+    $(TR $(TD $(LREF reverseArgs), $(LREF binaryReverseArgs))
         $(TD Predicate that reverses the order of its arguments.
     ))
     $(TR $(TD $(LREF toDelegate))
         $(TD Converts a callable to a delegate.
     ))
-    $(TR $(TD $(LREF unaryFun)), $(LREF binaryFun)
+    $(TR $(TD $(LREF unaryFun), $(LREF binaryFun))
```

Comment:
std.functional seems to be the only module with problems:

![image](https://cloud.githubusercontent.com/assets/4370550/23589633/e7fe441a-01d0-11e7-8a68-cbb0720e9429.png)